lib/src/mock_user.dart

```diff
@@ -74,18 +80,27 @@ class MockUser with EquatableMixin implements User {
   @override
   String? get refreshToken => _refreshToken;
 
+  set idTokenAuthTime(DateTime time) {
+    _idTokenAuthTime = time;
+  }
+
+  set idTokenExp(DateTime time) {
+    _idTokenExp = time;
+  }
+
   @override
   Future<String> getIdToken([bool forceRefresh = false]) {
-    var payload = {
+    final payload = {
       'name': displayName,
       'picture': photoURL,
       'iss': 'fake_iss',
       'aud': 'fake_aud',
-      'auth_time': 1655946582,
+      'auth_time': _idTokenAuthTime.millisecondsSinceEpoch ~/ 1000,
       'user_id': uid,
       'sub': uid,
-      'iat': 1656302136,
```

Comment:
I remove iat here because I found dart_jsonwebtoken will set iat.

Reply:
Thanks for the explanation. Would you mind adding a comment at line 120 to specify that `jwt.sign` will populate `iat`?

Reply:
Done